src/dotnet/CarbonAware.WebApi/Models/SerializableEmissionsData.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace CarbonAware.WebApi.Models;
+
+using CarbonAware.Model;
+using System.Text.Json.Serialization;
+
+[Serializable]
+public record SerializableEmissionsData
```

Comment:
I'm confused with all these 'serializable' classes and tests. Seems like there is extra code here that it is already managed by the .net framework in terms of serializing the return object(s). If the goal is to have formatted and/or rename properties, seems like it is possible just by setting a flag on the `Program.cs` like it is illustrated [here](https://docs.microsoft.com/en-us/aspnet/core/web-api/advanced/formatting?view=aspnetcore-6.0)

Reply:
Perhaps I could/should update the names to be `EmissionsDataDTO` for clarity around the pattern being used here, but this is standard convention to separate out Core domain objects from what is exposed to the user.  This enables a few things, but most importantly it separates our API spec from changes to the core.  The spec can change without touching the core (and everything else that relies on it) and the core can have things altered about it that never impact the spec (unless we want them to)

[Create Data Transfer Objects (DTOs)](https://docs.microsoft.com/en-us/aspnet/web-api/overview/data/using-web-api-with-entity-framework/part-5)

Reply:
Got it. One thing to consider is how much overhead (code mainteance + runtime) is added by just doing only data transformations. It is iterating over similar data or it is just a passthrough.